nextcloud-aio-helm-chart/templates/nextcloud-aio-nextcloud-deployment.yaml

```diff
@@ -173,7 +173,7 @@ spec:
               value: "{{ .Values.WHITEBOARD_ENABLED }}"
             - name: WHITEBOARD_SECRET
               value: "{{ .Values.WHITEBOARD_SECRET }}"
-          image: "{{ .Values.IMAGE_MIRROR_PREFIX }}{{ .Values.NEXTCLOUD_IMAGE_ORG }}/aio-nextcloud:20240925_080419"
+          image: "{{ .Values.IMAGE_MIRROR_PREFIX }}{{ .Values.NEXTCLOUD_IMAGE_ORG }}/aio-nextcloud:20240925_08041"
```

Comment:
todo: revert later on

Reply:
This needs to be added back as otherwise the update-helm.sh will not do anything. i changed this for testing purpose.